core/domain/user_domain.py

```diff
@@ -27,6 +27,8 @@
 
 from typing import Dict, List, Optional, TypedDict
 
+from core.domain import fs_services # pylint: disable=invalid-import-from # isort:skip
```

Comment:
Normally domain files shouldn't import anything, so this may cause you some trouble in the tests. And please do not add pylint-disables in your PRs -- they typically indicate that something is wrong and suppressing them will likely cause you further problems.

If a method here needs a services file then let's move that to user_services instead.

Reply:
Done